packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -221,6 +223,11 @@ class CupertinoTextField extends StatefulWidget {
   /// main text entry except a lighter font weight and a grey font color.
   final String placeholder;
 
+  /// The style to use for the placeholder text.
+  ///
+  /// Defaults to the [style] property with w300 font weight and grey color.
```

Comment:
Please mention in this doc that it's not just the style that's exclusively used but that it overrides the [style] parameter (which itself inherits from the theme).

Also mention whether it can be null and what does it mean when it is.